api/src/main/java/org/pmiops/workbench/workspaces/resources/UserRecentResourceServiceImpl.java

```diff
@@ -256,16 +193,12 @@ public List<DbUserRecentlyModifiedResource> findAllRecentlyModifiedResourcesByUs
   }
 
   /**
-   * Check number of entries in user_recently_modified_resource for user, If it exceeds
+   * Check number of entries in user_recently_modified_resource for user, If it exceeds or equals
```

Comment:
Clarifying the comment because this method actually reduces the list to (USER_ENTRY_COUNT - 1) so it should be run before adding a new one.

alt: we could update the logic so it runs after instead

Reply:
I can see 12 entries on the home page for recently modified, if that correct? 
<img width="1632" alt="Screen Shot 2022-07-19 at 2 40 19 PM" src="https://user-images.githubusercontent.com/34481816/179824800-1e25e165-c885-44fb-aa80-5775a23a0de0.png">

Reply:
It *should* be 10, yes.

But the logic for determining whether we call `handleLimit()` seemed a bit weird.  My guess is that this change will make it more consistent.